content/en/tracing/troubleshooting/agent_apm_resource_usage.md

```diff
@@ -0,0 +1,44 @@
+---
+title: APM Agent Resource Usage
+kind: Documentation
+aliases:
+  - /tracing/troubleshooting/agent_apm_resource_usage
+---
+
+## How the APM Agent works
+
+The APM Agent is CPU-bound and its CPU usage is correlated with the number of spans received per second.
+
+The APM Agent is designed to buffer unprocessed payloads in memory to avoid data loss. Consequently, throttling the Agent process because of an insufficient CPU limit can lead to out of memory.
+
+## Detect out-of-CPU
+
+The following metrics emitted by APM Agent can help detect out-of-CPU:
+- `datadog.trace_agent.cpu_percent`: CPU usage (in decicores), for example, 50 (half a core), 200 (two cores). It can be compared to the [maximum CPU percentage][1] configured for the APM Agent.
+- `datadog.trace_agent.ratelimit`: If lower than 1, it means payloads are being refused due to high resource usage (cpu or memory).
+
+See the full list of [APM Agent metrics][2].
+
+**Note:** If you disabled the [maximum CPU percentage for APM][1] in containerized environments, you can use the following metrics: `container.cpu.usage`, `container.cpu.limit` and `container.cpu.throttled`.
+
+
+[1]: /tracing/troubleshooting/agent_rate_limits/#maximum-cpu-percentage
```

Comment:
```suggestion
[1]: /tracing/troubleshooting/agent_rate_limits/#maximum-memory-limit
```
I would update like this so that it includes both the CPU and memory limits, or add this separately.